src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx

```diff
@@ -148,59 +148,64 @@ function Paginator({
 }: PaginatorProps) {
 	paginatedArray(numberOfPages, currentPage)
 	return (
-		<div className={s.paginator}>
-			{/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-			<a
-				className={classNames({ [s.disabled]: currentPage === 1 })}
-				onClick={(e) => {
-					e.preventDefault()
-					if (currentPage !== 1) {
-						onPageClicked(currentPage - 1)
-					}
-				}}
-			>
-				<IconArrowLeft16 />
-			</a>
-
-			{paginatedArray(numberOfPages, currentPage).map((pageNum, i) => {
-				if (pageNum === COLLAPSED) {
-					return (
-						// eslint-disable-next-line react/no-array-index-key
-						<a key={i} className={s.disabled}>
-							...
-						</a>
-					)
-				} else {
-					return (
-						// eslint-disable-next-line react/jsx-key, jsx-a11y/anchor-is-valid
-						<a
-							// eslint-disable-next-line react/no-array-index-key
-							key={i}
-							className={classNames({
-								[s.activePage]: currentPage === pageNum,
-							})}
-							onClick={(e) => {
-								e.preventDefault()
-								onPageClicked(pageNum as number)
-							}}
-						>
-							{pageNum}
-						</a>
-					)
-				}
-			})}
-			{/*eslint-disable-next-line jsx-a11y/anchor-is-valid*/}
-			<a
-				className={classNames({ [s.disabled]: currentPage === numberOfPages })}
-				onClick={(e) => {
-					e.preventDefault()
-					if (currentPage < numberOfPages) {
-						onPageClicked(currentPage + 1)
+		<nav className={s.paginator} aria-label="Pagination Navigation">
+			<ul>
+				<li>
+					<button
+						onClick={() => {
+							if (currentPage !== 1) {
+								onPageClicked(currentPage - 1)
+							}
+						}}
+						aria-label="Previous page"
+						disabled={currentPage === 1}
+						type="button"
+					>
+						<IconArrowLeft16 />
+					</button>
+				</li>
+
+				{paginatedArray(numberOfPages, currentPage).map((pageNum, i) => {
+					if (pageNum === COLLAPSED) {
+						return (
+							<li key={i} role="presentation">
+								<button disabled type="button">
+									...
+								</button>
+							</li>
+						)
+					} else {
+						return (
+							<li key={i}>
+								<button
+									onClick={() => {
+										onPageClicked(pageNum as number)
+									}}
+									aria-current={currentPage === pageNum ? 'page' : undefined}
+									aria-label={`Page ${pageNum} of ${numberOfPages}`}
```

Comment:
Setting the `aria-label` to something other than the visible label of the button will make the button unusable for voice control users. This is because voice commands are based off of the accessible name for an element, which in this case is computed using the `aria-label`. A voice control user would have to say something like "Click page 1 of 10" to activate the button, and they're likely to say "Click 1" based on the visible label. I think it's safe to not set an `aria-label` here since the context of total pages is close by.

Reply:
I'm less sure about the next/previous page buttons. I think we'd need user testing to see what people would most commonly call them, but I think they're fine as is for now.

Reply:
Just seeing these after I closed in favor or the other pagination component already in place, but looks like your comments apply there as well!